config/config.exs

```diff
@@ -115,7 +115,14 @@ config :oli, :recaptcha,
   secret: System.get_env("RECAPTCHA_PRIVATE_KEY")
 
 # Configure help
-config :oli, :help, dispatcher: Oli.Help.Providers.EmailHelp
+# HELP_PROVIDER env var must be a string representing an existing provider module, such as "EmailHelp"
+help_provider =
+  case System.get_env("HELP_PROVIDER") do
+    nil -> Oli.Help.Providers.EmailHelp
+    provider -> Module.concat([Oli, Help, Providers, provider])
```

Comment:
I did not know about `Module.concat`, very interesting.

Reply:
Actually, it is safer to use [Module.safe_concat/1](https://hexdocs.pm/elixir/1.12/Module.html#safe_concat/1) but in this case we can't since this file is loaded before any dependency and `safe_concat` raises an error if the module wasn't referenced yet.